src/AppInstallerCLICore/ExecutionProgress.cpp

```diff
@@ -19,7 +19,7 @@ namespace AppInstaller::CLI::Execution
 
         BytesFormatData s_bytesFormatData[] =
         {
-            // Multi-terabyate installers should be fairly rare for the foreseeable future...
+            // Multi-terabyte installers should be fairly rare for the foreseeable future...

```

Comment:
Oops. Thanks for catching this. Initial spell-checking passes are best effort, and a goal is for people to incrementally improve.